backend/routes/Department.go

```diff
@@ -0,0 +1,25 @@
+package routes
+
+import (
+	"net/http"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
+)
+
+const logTagDepartment = "[Department]"
```

Comment:
Department is not a great message, how about GetDepartment so we know the method

Reply:
I'm using the exact file name for all of them. Tell me if you think using GetDepartment (or the route names for all) is better